spec.emu

```diff
@@ -14,15 +14,14 @@ contributors: Richard Gibson
 <!-- es6num="24.3.2.2" -->
 <emu-clause id="sec-quotejsonstring" aoid="QuoteJSONString">
   <h1>Runtime Semantics: QuoteJSONString ( _value_ )</h1>
-  <p>The abstract operation QuoteJSONString with argument _value_ wraps a String value in QUOTATION MARK code units and escapes certain other code units within it.</p>
-  <p>As defined in the Unicode Standard, a <dfn id="high-surrogate-code-unit">high-surrogate code unit</dfn> is a code unit with a numeric value in the inclusive range 0xD800 to 0xDBFF and a <dfn id="low-surrogate-code-unit">low-surrogate code unit</dfn> is a code unit with a numeric value in the inclusive range 0xDC00 to 0xDFFF.</p>
+  <p>The abstract operation QuoteJSONString with argument _value_ wraps a String value in QUOTATION MARK code units and escapes certain other code <del>units</del><ins>points</ins> within it.</p>
```

Comment:
It’s a little weird to mix “code units” and “code points” in the same sentence, especially with the word “other“ in there:

> code units and escapes certain other code <del>units</del><ins>points</ins>